package.json

```diff
@@ -22,28 +22,26 @@
     "watch": "webpack --progress --colors --watch --watch-poll"
   },
   "devDependencies": {
-    "babel-core": "^6.23.1",
-    "babel-eslint": "^7.1.1",
-    "babel-loader": "^6.3.2",
-    "babel-polyfill": "^6.22.0",
-    "babel-preset-es2015": "^6.22.0",
-    "base64-loader": "^1.0.0",
+    "babel-core": "6.24.1",
+    "babel-eslint": "7.2.2",
+    "babel-loader": "6.4.1",
+    "babel-polyfill": "6.23.0",
+    "babel-preset-es2015": "6.24.1",
+    "base64-loader": "1.0.0",
     "copy-webpack-plugin": "^4.0.1",
     "docdash": "^0.4.0",
     "eslint": "^3.16.1",
     "eslint-config-scratch": "^3.1.0",
     "gh-pages": "^0.12.0",
     "hull.js": "0.2.10",
     "jsdoc": "^3.4.3",
-    "json": "9.0.4",
-    "json-loader": "^0.5.4",
-    "lodash.defaultsdeep": "4.6.0",
-    "raw-loader": "^0.5.1",
+    "json": "^9.0.4",
+    "raw-loader": "0.5.1",
     "scratch-render-fonts": "git+https://github.com/LLK/scratch-render-fonts.git",
     "tap": "^10.3.0",
     "travis-after-all": "^1.4.4",
     "twgl.js": "3.2.2",
-    "webpack": "^2.2.1",
```

Comment:
Similarly webpack will either work or not, and releases quite frequently. Although I don't trust them as much to not release breaking changes. Maybe the webpack-related packages should have `~`?  In our other repos, we pinned babel and webpack-related packages with `^` because the Greenkeeper spam was too much to keep up with (personally).